include/ninshiki_cpp/node/ninshiki_cpp_node.hpp

```diff
@@ -21,6 +21,12 @@
 #ifndef NINSHIKI_CPP__NODE__NINSHIKI_CPP_NODE_HPP_
 #define NINSHIKI_CPP__NODE__NINSHIKI_CPP_NODE_HPP_
 
+#include <memory>
+#include <opencv2/dnn.hpp>
+#include <opencv2/highgui.hpp>
+#include <opencv2/imgproc.hpp>
+#include <string>
```

Comment:
You don't need to change this, but please avoid changing stuffs that is not related with what you're trying to change (e.g. refactoring or formatting codes). It's better to create another pull request for that.